tests/data/test.txt

```diff
@@ -0,0 +1 @@
+Test txt file
```

Comment:
Have you considered using `conftest` to create a text file fixture? That way we don't need a "physical" file in the repo. Just an idea, what you have done here is perfect as is.